include/networkit/auxiliary/Log.hpp

```diff
@@ -6,49 +6,59 @@
 
 #include <networkit/auxiliary/StringBuilder.hpp>
 
+#include <tlx/define/deprecated.hpp>
+
 #ifdef _MSC_VER
-    #define NETWORKT_PRETTY_FUNCTION __FUNCSIG__
+#define NETWORKT_PRETTY_FUNCTION __FUNCSIG__
 #else
-    #define NETWORKT_PRETTY_FUNCTION __PRETTY_FUNCTION__
+#define NETWORKT_PRETTY_FUNCTION __PRETTY_FUNCTION__
 #endif // _MSC_VER
```

Comment:
This file has a lot of unrelated reformatting. Please remove these changes or factor them out into a commit of their own (that *only* does reformatting and nothing else).